### PR TITLE
Adds ability to quickly deploy server to Heroku with limitations

### PR DESCRIPTION
**Pull request by zol**
Limitations are SQLite and no websockets.

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import schema from './schema';
 
 let PORT = 3010;
 if (process.env.PORT) {
-  PORT = parseInt(process.env.PORT, 10) + 100;
+  PORT = parseInt(process.env.PORT, 10);
```
Comment (zol) [inline api/index.js]: I wasn't sure why the hardcoded magic number `100` was used here. Things seem to work without it and this change is required for Heroku.

**Comment by zol** [comment]
@stubailo @tmeasday - Just wanted to get this on your radar. These are the changes required to get one click deployments working on Heroku.

File: package.json [hunk 2]
```diff
@@ -40,6 +36,12 @@
     "nodemon": "^1.9.2"
   },
   "dependencies": {
+    "babel-cli": "^6.8.0",
```
Comment (tmeasday) [inline package.json]: Why is babel a dependency if it's running after the `build` step?

Comment (zol) [inline package.json]: Heroku installs `dependencies` when it runs the `postinstall` step in `package.json`. It's possible some of the `babel-*` dependencies aren't needed there, I didn't look too closely at that.

Comment (tmeasday) [inline package.json]: I'm confused by the documentation, but you are saying that it only installs production dependencies when building? This seems counter intuitive but I guess if that's what it does, this is what you have to do.

Comment (zol) [inline package.json]: The docs seem pretty clear to me. You can set an env variable `NPM_CONFIG_PRODUCTION` that's `true` by default to `false` if you want Heroku to install `devDependencies`, however they recommend you just move stuff to `dependencies` instead so you don't end up installing *all* the dev dependencies (e.g for linting) given you probably don't need them.

**Pull request closed without merging**